.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "resources/public/contracts/src/ens-repo"]
 	path = resources/public/contracts/src/ens-repo
 	url = https://github.com/ensdomains/ens.git
+[submodule "resources/public/contracts/src/strings-lib-repo"]
+	path = resources/public/contracts/src/strings-lib-repo
+	url = git@github.com:Arachnid/solidity-stringutils.git
```

Comment:
The strings library will cause problems very soon as there are breaking changes between solidity versions 0.4 and 0.5. We may need to look for an alternative or look at it in more detail.

Reply:
Do we need to migrate to solidity v0.5?

Reply:
But sure, whatever the reason. If this lib is incompatible we can get rid of it. But that will be addressed later when the time comes.

Reply:
Sure, I just wanted to mention it upfront if you saw someone upgraded it or something.
I'll also think about the solidity update if it caused too much problems but in general I think 0.5 is an improvement.